sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -72,8 +73,13 @@ class BigQueryEngineAdapter(InsertOverwriteWithMergeMixin):
                 exp.DataType.build("DATETIME", dialect=DIALECT),
             },
         },
-        support_coercing_compatible_types=True,
-    )
+        "support_coercing_compatible_types": True,
+    }
+    SCHEMA_DIFFER = SchemaDiffer(**SCHEMA_DIFFER_ARGS)
+
+    @classproperty
+    def schema_differ_args(cls) -> t.Dict[str, t.Any]:
+        return cls.SCHEMA_DIFFER_ARGS
```

Comment:
Why not just do `cls.SCHEMA_DIFFER_ARGS` instead of creating this property which then does that?